crates/sui-rosetta/src/operations.rs

```diff
@@ -259,7 +292,7 @@ impl Operations {
                 }
                 SuiArgument::GasCoin => (),
                 // Might not be a SUI coin
-                SuiArgument::Input(_) => return None,
+                SuiArgument::Input(_) => (),
```

Comment:
why is this change needed?

Reply:
Without this change, in case a pay (coin, other than SUI) operation, this method return early at this line without the required amount which is calculated a few lines below
```
 let value: u64 = match *amount {
                        SuiArgument::Input(i) => {
                            u64::from_str(inputs[i as usize].pure()?.to_json_value().as_str()?)
                                .ok()?
                        }
                        ...
```